packages/@n8n/benchmark/src/scenario/scenario-loader.ts

```diff
@@ -18,6 +18,9 @@ export class ScenarioLoader {
 		const scenarios: Scenario[] = [];
 
 		for (const folder of scenarioFolders) {
+			if (filter && !folder.toLowerCase().includes(filter.toLowerCase())) {
+				continue;
+			}
```

Comment:
Could we maybe make this a regex at some point?

Reply:
I thought about allowing glob pattern for it, but at least for now the use case would be to filter only a single scenario, I didn't want to complicate it. Let's revisit this when there's a need for this

Reply:
Yeah, that's fine. I just know I use the partial matches in Jest all the time, so if I was running benchmarks, I'd probably want that as well.